modules/framework/galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -1938,7 +1938,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RBACRolesSummary'
+                $ref: '#/components/schemas/RBACRole'
```

Comment:
The success response should return a list/array of roles (currently, it looks like it'll return one role)

Reply:
right. oops.